### PR TITLE
KILL THE GLOBALS 🗡️🌎

### PR DESCRIPTION
**Pull request by autodidaddict**
This PR takes everything that used to be defined within a `lazy_static` and moves it into a field available as part of the `WasccHost` struct. There are a number of benefits to this, not the least of which is cleaner code. This also means that two hosts can theoretically exist within the same parent process, each with their own private routing table.

Additionally, the `Actor::from_gantry` function has been moved into the `WasccHost` struct. It makes more sense for this to be part of the host (which is where the gantry configuration lives) rather than part of the concept of an isolated `Actor`. The new method is `add_actor_from_gantry`, and it takes the actor's `subject` (primary key), pulls from Gantry, and then adds it to the host. 

This also has an added benefit of moving the auth hook invocation and token verification into the host rather than the `Actor` struct, which I think is a more appropriate separation of concerns.

Those evaluating this PR - please run this code in your environment to ensure there are no regressions. All tests currently pass, but I want to make sure there are no copy/paste/refactor failures that only show up in edge cases.

### CODE REVIEW

**Comment by ewbankkit** [comment]
Does this close https://github.com/wascc/wascc-host/issues/13?

**Comment by autodidaddict** [comment]
Yes

File: src/lib.rs [hunk 1]
```diff
@@ -381,25 +482,20 @@ impl WasccHost {
...
         if actor.len() == 56 && actor.starts_with('M') {
             // This is an actor's public subject
-            self.add_actor(Actor::from_gantry(&actor)?)
+            self.add_actor_from_gantry(actor)
         } else {
             self.add_actor(Actor::from_file(&actor)?)
         }
     }
 
-    #[cfg(not(feature = "gantry"))]
```
Comment (ewbankkit) [inline src/lib.rs:390]: `add_actor_from_path` is still called from `apply_manifest` if I compile with just ` features = ["manifest"]` (i.e. no `gantry` feature).

**Comment by autodidaddict** [comment]
Add actor from path should be called as a fallback when gantry isn't enabled

**Comment by ewbankkit** [comment]
Apart from the one compilation issue that will require a change to `WasccHost.apply_manifest`, compiling and running with https://github.com/wascc/aws-lambda-wascc-runtime was fine.

**Comment by ewbankkit** [comment]
Verified that compiling (and running) with only the `manifest` feature is OK.

**Review by ewbankkit — APPROVED** [review]
LGTM.
